lte/gateway/c/oai/tasks/mme_app/mme_app_state_manager.cpp

```diff
@@ -259,5 +266,54 @@ int MmeNasStateManager::read_ue_state_from_db() {
   return RETURNok;
 }
 
+void MmeNasStateManager::create_mme_ueip_imsi_map() {
+  if (!persist_state_enabled) {
+    OAILOG_ERROR(log_task, "persist_state_enabled is not enabled \n");
+    return;
+  }
+  ueip_imsi_map = new UeIpImsiMap;
+  if (!ueip_imsi_map) {
+    OAILOG_CRITICAL(log_task, "Failed to allocate memory for ueip_imsi_map \n");
+    return;
+  }
+  oai::MmeUeIpImsiMap ueip_proto = oai::MmeUeIpImsiMap();
+  redis_client->read_proto(MME_UEIP_IMSI_MAP_NAME, ueip_proto);
+
+  MmeNasStateConverter::mme_app_proto_to_ueip_imsi_map(
+      ueip_proto, ueip_imsi_map);
+  return;
+}
+
+void MmeNasStateManager::write_mme_ueip_imsi_map_to_db() {
+  if (!persist_state_enabled) {
+    OAILOG_ERROR(log_task, "persist_state_enabled is not enabled \n");
+    return;
+  }
+
+  oai::MmeUeIpImsiMap ueip_proto = oai::MmeUeIpImsiMap();
+  MmeNasStateConverter::mme_app_ueip_imsi_map_to_proto(
+      ueip_imsi_map, &ueip_proto);
+  std::string proto_msg;
+  redis_client->serialize(ueip_proto, proto_msg);
+  std::size_t new_hash = std::hash<std::string>{}(proto_msg);
+
+  // s1ap_imsi_map is not state service synced, so version will not be updated
```

Comment:
fix the comment

Reply:
Version is not updated, so added this comment. Could you please elaborate on required comment